backend/src/main/java/net/vidux/camhub/discovery/RawCameraDataProcessing.java

```diff
@@ -0,0 +1,28 @@
+package net.vidux.camhub.discovery;
+
+import net.vidux.camhub.camera.Camera;
+import net.vidux.camhub.camera.CameraRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+
+@Component
+class RawCameraDataProcessing {
+
+  @Autowired
+  CameraFactory cameraFactory;
+
+  @Autowired
+  CameraRepository cameraRepository;
```

Comment:
Inkább preferáljuk a konstruktoron keresztül történő injektálást, amihez nincs szükség reflection-re:

```suggestion
  private CameraFactory cameraFactory;
  private CameraRepository cameraRepository;
  
  @Autowired
  public RawCameraDataProcessing(CameraFactory cameraFactory, CameraRepository cameraRepository) {
    this.cameraFactory = cameraFactory;
    this.cameraRepository = cameraRepository;
  }
```